gobblin-utility/src/main/java/org/apache/gobblin/util/PullFileLoader.java

```diff
@@ -212,6 +212,47 @@ public Config loadPullFile(Path path, Config sysProps, boolean loadGlobalPropert
     }
   }
 
+  public List<Path> fetchJobFilesRecursively(Path path) {
+    return getSortedPaths(fetchJobFilePathsRecursivelyHelper(path));
+  }
+
+  private List<Path> getSortedPaths(List<PathWithTimeStamp> pathsWithTimeStamps) {
+    List<Path> sortedPaths = Lists.newArrayList();
+    Collections.sort(pathsWithTimeStamps, Comparator.comparingLong(o -> o.timeStamp));
+    for (PathWithTimeStamp pathWithTimeStamp : pathsWithTimeStamps) {
+      sortedPaths.add(pathWithTimeStamp.path);
+    }
+    return sortedPaths;
+  }
+
+  private List<PathWithTimeStamp> fetchJobFilePathsRecursivelyHelper(Path path) {
```

Comment:
See if this can be reused to replace `loadPullFilesRecursivelyHelper`

Reply:
Resolved